app/gui2/src/main.ts

```diff
@@ -1,38 +1,76 @@
+import { options as appOptions } from '@/util/appOptions'
+import { isDevMode } from '@/util/detect'
+import { urlParams } from '@/util/urlParams'
+import {
+  checkMinSupportedVersion as checkMinimumSupportedVersion,
+  displayDeprecatedVersionDialog,
+} from '@/util/version'
+import { run as runDashboard } from 'enso-authentication'
+import { isOnLinux } from 'enso-common/src/detect'
 import 'enso-dashboard/src/tailwind.css'
+import { decodeQueryParams } from 'lib0/url'
 
-const INITIAL_URL_KEY = `Enso-initial-url`
+const vueAppEntry = import('./createApp')
 
-import * as dashboard from 'enso-authentication'
-import { isMac } from 'lib0/environment'
-import { decodeQueryParams } from 'lib0/url'
+const INITIAL_URL_KEY = `Enso-initial-url`
+const SCAM_WARNING_TIMEOUT = 1000
 
 const params = decodeQueryParams(location.href)
-
-// Temporary hardcode
-const config = {
-  supportsLocalBackend: true,
-  supportsDeepLinks: isMac,
-  shouldUseAuthentication: false,
-  projectManagerUrl: PROJECT_MANAGER_URL,
-  initialProjectName: params.project ?? null,
-}
+const initialProjectName = params.project ?? null
```

Comment:
This is now incorrect, and in fact impossible to use. The `startup.project` config option should be used instead.